src/integrationTest-samples/csharp-plain-sam-app/template.yaml

```diff
@@ -0,0 +1,40 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+    Test SAM Template
+
+# More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
+Globals:
+    Function:
+        Timeout: 10
+
+Resources:
+    HelloWorldFunction:
+        Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+        Properties:
+            CodeUri: ./src/HelloWorld/
+            Handler: HelloWorld::HelloWorld.Function::FunctionHandler
+            Runtime: dotnetcore2.1
```

Comment:
Is it worth making the folder something like "csharp-plain-sam-app-21", so that a net core 3.1 sample can be slotted in beside this?

Reply:
yep. lots of things can be added at low-cost yet high test-coverage, because of the separation of "config" vs "execute" phase: most important logic happens in the "config" phase, unlike legacy code which made config-related decisions throughout the codepath.